axonserver/pom.xml

```diff
@@ -415,7 +415,23 @@
                     </execution>
                 </executions>
             </plugin>
-
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>1.8.0</version>
+                <configuration>
+                    <container>
+                        <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
+                    </container>
+                    <from>
+                        <image>gcr.io/distroless/java:11</image>
+                    </from>
+                    <to>
+                        <image>axoniq/axon-server</image>
```

Comment:
Is this image intended for (local) testing? Then it should be "`eu.gcr.io/axoniq-devops/axonserver`". The Docker Hub image is "`axoniq/axonserver`" without a dash.